### PR TITLE
What we do pages

### PR DESCRIPTION
**Pull request by VladimirZaets**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://github.com/hlxsites/merative/issues/126

Test URLs:
- Before: 

https://main--merative2--hlxsites.hlx.page/drafts/vzaets/healthcare-analytics
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-decision-support
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-development
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-development/case-studies
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/social-program-management
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/social-program-management/case-studies
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/real-world-evidence
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/merge-imaging
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/merge-imaging/imaging-case-studies
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/cardiology
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/radiology
https://main--merative2--hlxsites.hlx.page/drafts/vzaets/partner-innovations

- After: 
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/healthcare-analytics
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-decision-support
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-development
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-development/case-studies
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/social-program-management
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/social-program-management/case-studies
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/real-world-evidence
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/merge-imaging
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/merge-imaging/imaging-case-studies
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/cardiology
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/radiology
https://author-bio-multiple--merative2--hlxsites.hlx.page/drafts/vzaets/partner-innovations

### CODE REVIEW

File: blocks/linkable-cards/linkable-cards.css [hunk 1]
```diff
@@ -17,17 +17,64 @@
...
+.linkable-cards > div::after {
+  content: "";
+    display: inline-block;
+    height: 55px;
+    right: 20px;
+    bottom: 20px;
+    position: absolute;
+    width: 55px;
+    background: url("../../icons/download-arrow-white.svg") no-repeat center;
+    border: 2px solid #fff;
```
Comment (arumsey) [inline blocks/linkable-cards/linkable-cards.css:36]: could use a var for color

File: blocks/linkable-cards/linkable-cards.css [hunk 2]
```diff
@@ -17,17 +17,64 @@
...
+    border-radius: 50%;
+}
+
+.linkable-cards.neutral-bone-style>div::after {
+  background: url("../../icons/download-arrow-purple.svg") no-repeat center;
+  border: 2px solid var(--primary-purple);
+}
+
+.linkable-cards>div:hover {
```
Comment (arumsey) [inline blocks/linkable-cards/linkable-cards.css:45]: add space around `>`

File: blocks/linkable-cards/linkable-cards.css [hunk 3]
```diff
@@ -17,17 +17,64 @@
...
+  transition: all .2s ease-out;
+  box-shadow: 2px 4px 4px 2px rgb(0 0 0 / 15%);
+}
+
+.section .linkable-cards>div:hover::after {
```
Comment (arumsey) [inline blocks/linkable-cards/linkable-cards.css:50]: add space around `>`

File: styles/styles.css [hunk 4]
```diff
@@ -267,12 +267,87 @@ blockquote {
...
+  max-width: 350px;
+}
+
+main .columns.columns-gap-50>div {
   gap: 50px
 }
 
-main .column-content-top-border div>div {
-  margin-bottom: 32px
+main .padding-bottom-40 {
```
Comment (arumsey) [inline styles/styles.css:279]: These are overly technical section styles an author will need to know and pick from to render out a specific design. Can authors pick and choose or are only certain combainations of padding allowed? Could terms such as `compact` or `spacious` be used to indicate how much padding should be applied to a section?

File: styles/styles.css [hunk 5]
```diff
@@ -267,12 +267,87 @@ blockquote {
...
+main .section.margin-top-40,
+main .section .columns.margin-top-40 {
+  margin-top: 40px;
+}
+
+main .bg-neutral-bone {
+  background-color: var(--neutral-bone);
+}
+
+main .section.notification-section {
```
Comment (arumsey) [inline styles/styles.css:309]: The string `section` in `notification-section` redundant. Could the additional section style just be called `notification`?

File: styles/styles.css [hunk 6]
```diff
@@ -267,12 +267,87 @@ blockquote {
...
+main .section.notification-section>div>p::before {
+  content: '';
+  display: inline-block;
+  width: 5px;
+  height: 32px;
+  margin: -9px 50px -7px 0; 
+  background-image: var(--gradient-diagonal-down);
+}
+
+main .section.what-we-do>.default-content-wrapper>h4 {
```
Comment (arumsey) [inline styles/styles.css:323]: Now we're getting very specific. This style will noly ever be used with `what-we-do` section?

**Review by arumsey — COMMENTED** [review]
Some naming comments I think are worth dicussing. Also, please be consistent with spacing around `>` in CSS.

**Review by venturewise — APPROVED** [review]
I will merge this so that I can correct any inconsistencies I brought on by merging my earlier PR before I noticed this was still open in the next PR. My apologies for complicating things.

**Comment by venturewise** [comment]
I will fix any leftover linting problems in my next PR.

**Pull request merged**